src/core/stores/VNode.ts

```diff
@@ -453,6 +453,32 @@ export class VNode {
             return next;
         }
     }
+    /**
+     * Return the previous leaf in a depth-first pre-order traversal of the
+     * tree that satisfies the given predicate. If no predicate is given return
+     * the previous leaf in a depth-first pre-order traversal of the tree.
+     * A leaf is a childless node.
```

Comment:
If the guy does not know what a tree "leaf" is in programming, then it's highly unlikely he would have understood "depth-first pre-order traversal" or "predicate" anyway, just sayin'... ;-) (same for `nextLeaf`)